index.html

```diff
@@ -51,14 +51,37 @@ <h3>Site Preview</h3>
         <!-- Mary is making cards for testimonials here -->
         <section id="testimonials">
             <h3>Testimonials</h3>
+
+        <div class=Container></div>
+
             <div class="cards"></div>
-            <div class="row"></div>
-            <div class="person"></div>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque quibusdam aliquam inventore fuga porro
-                dicta perspiciatis quia nihil tempora quos odio corrupti, eius, distinctio error quaerat itaque
-                voluptatem atque reiciendis. Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque quibusdam
-                aliquam inventore fuga porro dicta perspiciatis quia nihil tempora quos odio corrupti, eius, distinctio
-                error quaerat itaque voluptatem atque reiciendis.</p>
+                <div class="nameHere"></div>
```

Comment:
If you're testing how this looks on the HTML page, you should add some contents inside of the div. Right now, you applied a class to an empty div, so you won't see anything on the HTML page to see if the class is working